composer.json

```diff
@@ -52,7 +52,9 @@
         "monolog/monolog": "~1.11",
         "phpunit/phpunit": "~4.5",
         "satooshi/php-coveralls": "~0.6",
-        "symfony/phpunit-bridge": "~2.8|~3.0"
+        "symfony/phpunit-bridge": "~2.8|~3.0",
+        "doctrine/orm": "^2.4.8",
```

Comment:
This is in the unit test, therefore the dependency is ok in `require-dev`.
Normal installs will not get it pulled, as dev requirements are only pulled for the root project.
Therefore, everything good here. :)